input/fsh/at-core-ext-patient-religion.fsh

```diff
@@ -19,7 +19,7 @@ Description:  "HL7 Austria FHIR Core Extension for the religion (registered in A
 * extension[code].value[x] from HL7AustriaReligionVS (extensible)
 //* extension[code].value[x] ^binding.valueSetUri = "urn:oid:2.16.840.1.113883.2.16.1.4.1"
 * extension[code].value[x].coding.system ^fixedUri = "urn:oid:2.16.840.1.113883.2.16.1.4.1"
-* extension[code].value[x].coding.system ^short = "OID for the HL7 AT ReligionAustria ValueSet"
+* extension[code].value[x].coding.system ^short = "OID for the HL7 AT codeSystem used by the ELGA valueSet for religion"

```

Comment:
ValueSet -> Typo

Reply:
Typos are corrected now with the latest commit on this issue branch